.github/static/environment_forge_full.yml

```diff
@@ -24,22 +26,21 @@ dependencies:
  - packaging>=20.0
  - ipywidgets>=7.5.0
  - broadbean>=0.9.1
+ # test requirements from setup.cfg
  - deepdiff>=5.0.2
- - pyqtgraph
- - pyqt
- - QtPy
  - hypothesis>=5.49.0
  - pytest>=6.0.0
  - pytest-rerunfailures>=5.0.0
  - pytest-xdist>=2.0.0
  - pytest-mock>=3.0.0
- - pyzmq
  - lxml>=4.3.0
  - gitpython>=3.0.6
- - testpath
+ # extra relevant requirements from setup.cfg
+ - pyqtgraph>=0.11.0
+ # required for building docs
  - pandoc
```

Comment:
We are not currently building the docs with Anaconda so we could remove this

Reply:
indeed, let me remove it